src/lib/StyleButton.js

```diff
@@ -17,7 +17,9 @@ export default class StyleButton extends Component {
   }
 
   render(): React.Element {
-    let {style, ...otherProps} = this.props;
+    /* eslint-disable no-unused-vars */
+    let {style, onToggle, ...otherProps} = this.props;
```

Comment:
Better to do:
```
let {style, onToggle, ...otherProps} = this.props; // eslint-disable-line no-unused-vars
```
.. although it would be better if eslint could just properly handle object rest de-structuring. (I thought they fixed that before, evidently not.)